<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Task_pointClass</class>
 <widget class="QMainWindow" name="Task_pointClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Task_point</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="addPoint">
        <property name="text">
         <string>addPoint</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removePoint">
        <property name="text">
         <string>removePoint</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="m_ctlFilter"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="columnCount">
       <number>6</number>
      </property>
      <attribute name="headerDefaultSectionSize">
       <number>150</number>
      </attribute>
      <column>
       <property name="text">
        <string>id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>名称</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>类型</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>语音编号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>目标点x</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>目标点y</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>847</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="task_point.qrc"/>
 </resources>
 <connections/>
</ui>
